src/core/ruby_version.rs

```diff
@@ -919,6 +919,88 @@ macro_rules! get_stack_frame_2_5_0(
     )
 );
 
+macro_rules! get_stack_frame_3_3_0(
```

Comment:
The classext approach used here is only valid in ruby 3.3.0+, it may be possible to get the info in another way in earlier versions, but the struct i'm using wasn't added until 3.3.0